roles/certbot-cloudflare/templates/cloudflare.ini.tpl

```diff
@@ -0,0 +1 @@
+dns_cloudflare_api_token={{ cloudflare_api_token }}
```

Comment:
Where does this token get populated from?

Reply:
@mplorentz: It would get populated from vars passed in through an inventory file-- either inline  or as a separate var file.  There is an example of these variables in the full nginx server task (#28): [vars/main.yaml](https://github.com/planetary-social/ansible-scripts/blob/cfd3c36226001af1108dd1d5366cfe2d7b7b2f92/vars/main.yaml).

The api token could be generated by someone in the planetary org on cloudflare and then they'd populate all their secrets in their local inventory file and run the proper playbook.